lib/Semantics/resolve-names.cpp

```diff
@@ -375,10 +382,9 @@ class ImplicitRulesVisitor : public DeclTypeSpecVisitor {
 protected:
   void BeginScope(const Scope &);
   void SetScope(const Scope &);
+  void SetImplicitRules(ImplicitRules &rules) { implicitRules_ = &rules; }
```

Comment:
This isn't used.

Reply:
Thanks (I didn't clean it out after it became needless).